content/posts/2022-01-03-scm-manager-plugin-development/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: SCM-Manager - Recap 2021
+date: 2022-01-03T10:00:00+0000
+author: Eduard Heimbuch
+categories:
+- scm-manager
+- release
+keywords:
+- recap
+- plugin
+- development
+- guide
+- tutorial
+---
+
+Hey SCM-Manager Community,
+
+It's the first monday of the new year, and we just started working on the next major features to improve your SCM-Manager experience.
+But before going forward we wanted to do a little recap and share our highlights regarding SCM-Manager of 2021.
+
+## Recap
+We had 16 minor releases in the last year including these big features/changes:
+- Repository Import and Export for Git, HG and Subversion
+- Frontend rebuild with react-query which dramatically improved the performance
+- Metrics for SCM-Manager which can be collected with the related plugins as [JSON](https://scm-manager.org/plugins/scm-metrics-json-plugin/) or [PromQL](https://scm-manager.org/plugins/scm-metrics-prometheus-plugin/)
```

Comment:
The plugin does not collect the metrics as promql. PromQL is the query language of prometheus. The format is called [OpenMetrics](https://openmetrics.io/).